src/support/ResourcePolicies.test.tsx

```diff
@@ -0,0 +1,118 @@
+jest.mock(
+  'src/libs/error',
+  (): ErrorExports => ({
+    ...jest.requireActual('src/libs/error'),
+    reportError: jest.fn(),
+  })
+);
+
+describe('ResourcePolicies', () => {
+  function setGetResourcePoliciesMock(getResourcePolicies: jest.Mock<Promise<Awaited<object>>, []>) {
+    asMockedFn(Ajax).mockImplementation(() => {
+      return {
+        SamResources: { getResourcePolicies } as Partial<ReturnType<typeof Ajax>['SamResources']>,
+      } as ReturnType<typeof Ajax>;
+    });
+  }
+
+  it('calls Ajax().SamResources.getResourcePolicies and displays the result', async () => {
+    // Arrange
+    const testValue = uuidv4();
+    const getResourcePolicies = jest.fn(() => Promise.resolve({ policy: testValue }));
+    setGetResourcePoliciesMock(getResourcePolicies);
+    const fqResourceId: FullyQualifiedResourceId = { resourceId: 'resource-id', resourceTypeName: 'resource-type' };
+    const props: ResourceTypeSummaryProps = {
+      displayName: 'display-name',
+      fqResourceId,
+      loadSupportSummaryFn: undefined,
+    };
+
+    // Act
+    await act(async () => {
+      render(<ResourcePolicies {...props} />);
+    });
+
+    // Assert
+    expect(getResourcePolicies).toHaveBeenCalledWith(fqResourceId);
+    expect(screen.getByText('Sam Policies')).toBeInTheDocument();
+    expect(screen.getByText(new RegExp(testValue, 'i'))).toBeInTheDocument();
+  });
+
+  it('displays an error message when getResourcePolicies throws an error', async () => {
+    // Arrange
+    const errorMessage = 'test error message';
+    const getResourcePolicies = jest.fn(() => Promise.reject(new Error(errorMessage)));
+    setGetResourcePoliciesMock(getResourcePolicies);
+    const fqResourceId: FullyQualifiedResourceId = { resourceId: 'resource-id', resourceTypeName: 'resource-type' };
+    const props: ResourceTypeSummaryProps = {
+      displayName: 'display-name',
+      fqResourceId,
+      loadSupportSummaryFn: undefined,
+    };
+
+    // Act
+    await act(async () => {
+      render(<ResourcePolicies {...props} />);
+    });
+
+    // Assert
+    expect(getResourcePolicies).toHaveBeenCalledWith(fqResourceId);
+    expect(reportError).toHaveBeenCalled();
+  });
+
+  it('displays an error message when getResourcePolicies returns an empty array', async () => {
+    // Arrange
+    const getResourcePolicies = jest.fn(() => Promise.resolve([]));
+    setGetResourcePoliciesMock(getResourcePolicies);
+    const fqResourceId: FullyQualifiedResourceId = { resourceId: 'resource-id', resourceTypeName: 'resource-type' };
+    const props: ResourceTypeSummaryProps = {
+      displayName: 'display-name',
+      fqResourceId,
+      loadSupportSummaryFn: undefined,
+    };
+
+    // Act
+    await act(async () => {
+      render(<ResourcePolicies {...props} />);
+    });
+
+    // Assert
+    expect(getResourcePolicies).toHaveBeenCalledWith(fqResourceId);
+    expect(screen.getByText('No policies found')).toBeInTheDocument();
```

Comment:
Nit: `getBy...` queries throw an error if the element is not found, so `expect(...).toBeInTheDocument()` isn't necessary here. But it may also be a style preference.

```suggestion
    screen.getByText('No policies found');
```